src/services/EcoBridge/EcoBridge.hooks.ts

```diff
@@ -303,7 +303,7 @@ export const useBridgeCollectHandlers = () => {
     [dispatch]
   )
 
-  const collectableCurrency = useBridgeToken(collectableTx?.assetAddressL1, collectableTx?.toChainId ?? 0)
+  const collectableCurrency = useBridgeToken(collectableTx?.assetAddressL1, collectableTx?.fromChainId ?? 0)
```

Comment:
When transaction comes from subgraph I cannot pair `toToken` but I always have `fromToken`. Currently bridging is 1-1 e.g `USDC`->`USDC` so it doesn't matter which address I pass to `hook` because the symbol, icon, name of currency are the same